File: Networking/src/main/java/request/GetParticipantsForEventWithCountRequest.java
package request;

public class GetParticipantsForEventWithCountRequest implements Request {
    private final Long eventId;

    public GetParticipantsForEventWithCountRequest(long eventId) {
        this.eventId = eventId;
    }

    public Long getEventId() {
        return eventId;
    }
}

File: Networking/Request/GetParticipantsForEventWithCountRequest.cs
using System.Text.Json.Serialization;

namespace Networking.Request
{

    public class GetParticipantsForEventWithCountRequest: IRequest
    {
        [JsonPropertyName("eventId")]
        public long EventId { get; set; }

        [JsonConstructor]
        public GetParticipantsForEventWithCountRequest()
        {

        }

        public GetParticipantsForEventWithCountRequest(long eventId)
        {
            EventId = eventId;
        }
}
}